components/gcp-click-to-deploy/src/DeployForm.tsx

```diff
@@ -609,9 +618,9 @@ export default class DeployForm extends React.Component<any, DeployFormState> {
               this._appendLine('your kubeflow app url should be ready within 20 minutes (by '
                 + readyTime.toLocaleTimeString() + '): https://'
                 + this.state.deploymentName + '.endpoints.' + this.state.project + '.cloud.goog');
+              clearInterval(monitorInterval);
+              this._redirectToKFDashboard(dashboardUri);
             }
-            clearInterval(monitorInterval);
```

Comment:
We might want to do clearInterval() for both cases (skip or not skip).

Reply:
thanks. makes sense.